Applications/IRCClient/IRCWindow.cpp

```diff
@@ -111,6 +111,26 @@ IRCWindow::IRCWindow(IRCClient& client, void* owner, Type type, const String& na
                 m_client.handle_devoice_user_action(m_name.characters(), nick.characters());
             }));
 
+            m_context_menu->add_action(GUI::Action::create("Hop", [&](const GUI::Action&) {
+                GUI::ModelIndex new_index = member_view.selection().first();
+                auto nick = member_view.model()->data(new_index, IRCChannelMemberListModel::Role::Display).to_string();
```

Comment:
This logic could be moved to a helper function since it's repeated a bunch of times :)

Reply:
Yes, yes it could. Seems like that should be a separate PR...

Reply:
Or just a separate commit before this one